### ISSUE TITLE
Bower, Grunt (Yeoman) Installation

### ISSUE THREAD

**Issue by wonderment** [ericam/susy#423]
I'm confused by the installation for Susy in general. The headings are all the same on the Getting Started page, so I'm not sure whether I actually need to run the gem install command at the top or not. Presumably, if I'm using Bower with Yeoman/Grunt it's not necessary, correct?

I'm using Yeoman's web-app generator and running:

```
bower install susy --save-dev
```

drops it into bower_components just fine. Adding:

```
@import "../../bower_components/susy/sass/susy";
```

doesn't give me any output.

Aside from Bower, I've tried Grunt by using:

```
npm install susy@beta --save-dev
```

which drops it into node_modules just fine but using:

```
sass: {
      options: {
        loadPath: 'bower_components',
        require: 'susy'
      }

...
```

in Gruntfile.js simply returns a "Warning: LoadError: cannot load such file -- susy"

I've run out of ideas. I appreciate any help you can offer.

Thank you!

**Comment by aaronagray** [comment]
There are two main steps to work with Sass / Susy.
1. Include Susy via `@import`
2. Compile Susy along with the rest of your Scss.

Susy needs to be compiled with LibSass or Ruby Sass. If you are using a framework like Meteor, Ember, or Rails, then the compile mechanism may be dictated for you. For instance, Meteor uses node-sass (which uses LibSass) while Rails uses sass-rails, which requires Ruby Sass.

Outside of a framework, you may use something like Grunt or Gulp to include and configure the compile step. In this scenario, the easiest way to get started is to install & use `node-sass` & install Susy locally with Bower. The specifics will depend on your own workflow.

A few things could be going on if you are installing Susy and simply getting no output.
1. You're not using Susy after importing it. Susy doesn't output anything by default, so you need to call one of the Susy mixins to confirm it's working.
1. You are including Susy properly, but Grunt / Gulp isn't compiling it, which would require configuration specific to your environment.

Hope this helps.

**Comment by wonderment** [comment]
Hi Aaron,

Thank you so much for your thorough response. I have been compiling other Scss projects, so it wasn't an issue in that regard.

I didn't realize it wouldn't render any output in the compiled CSS file unless the mixins were called (d'oh).

For those using Yeoman's webapp and want to utilize Bower, I ran the following:

```
$ yo webapp (selecting SASS option only)
$ bower install susy --save-dev
```

For some reason, the current webapp generator which typically drops in bower dependencies into the main.scss file (wiredep) doesn't care about the susy bower package so I added:

```
// bower:scss
// endbower
@import "susy/sass/susy";
```

followed by the typical "$susy:" settings and from there, it worked just fine -- pretty simple in hindsight.

I'll close it out, so I can now play around with Susy! ;-)

Thanks again.
